src/components/Organizations/Users/index.js

```diff
@@ -0,0 +1,213 @@
+
+import useSortableData from '../../../lib/withSortedItems';
+import AddUserToOrganization from '../AddUserToOrganization';
+import PaginatedTable from '../PaginatedTable/PaginatedTable';
+import { Footer, TableActions } from '../SharedStyles';
+import { StyledUsers } from './Styles';
+
+export const getLinkData = (userSlug, organizationSlug, organizationName) => ({
+  urlObject: {
+    pathname: '/organizations/user',
+    query: { userSlug, organizationSlug: organizationSlug, organizationName: organizationName },
+  },
+  asPath: `/organizations/${organizationSlug}/users/${userSlug}`,
+});
+const DELETE_USER = gql`
+  mutation removeUserFromOrganizationGroups($organization: Int!, $email: String!) {
+    removeUserFromOrganizationGroups(input: { user: { email: $email }, organization: $organization }) {
+      id
+    }
+  }
+`;
+
+/**
+ * The primary list of users.
+ */
+const Users = ({ users = [], organization, organizationId, organizationName, refetch }) => {
+  const [userModalOpen, setUserModalOpen] = useState(false);
+  const [selectedUser, setSelectedUser] = useState('');
+
+  const [addUserModalOpen, setAddUserModalOpen] = useState(false);
+  const [deleteUserModalOpen, setDeleteUserModalOpen] = useState(false);
+
+  const [dynamicUsers, setDynamicUsers] = useState(users);
+
+  const closeUserModal = () => {
+    setSelectedUser('');
+    setUserModalOpen(false);
+  };
+
+  useEffect(() => {
+    setDynamicUsers(users);
+  }, [users]);
+
+  const { sortedItems } = useSortableData(dynamicUsers, {
+    key: 'email',
+    direction: 'ascending',
+  });
+
+  useEffect(() => {
+    if (sortedItems) {
+      setDynamicUsers(sortedItems);
+    }
+  }, [sortedItems]);
+
+  const UsersColumns = [
+    {
+      width: '15%',
+      key: 'firstName',
+      render: ({ firstName }) => {
+        return firstName ? <div className="name">{firstName}</div> : <>First name - </>;
+      },
+    },
+    {
+      width: '15%',
+      key: 'lastName',
+      render: ({ lastName }) => {
+        return lastName ? <div className="lastname">{lastName}</div> : <>Last name -</>;
+      },
+    },
+    {
+      width: '25%',
+      key: 'email',
+      render: ({ email }) => {
+        const linkData = getLinkData(email, organizationId, organizationName);
+        return (
+          <div className="email">
+            <Link href={linkData.urlObject} as={linkData.asPath}>
+              <a>{email}</a>
+            </Link>
+          </div>
+        );
+      },
+    },
+    {
+      width: '15%',
+      key: 'groups',
+      render: () => {
+        return organization ? <div className="groups">Groups: {organization.groups.length}</div> : <>Groups -</>;
+      },
+    },
+    {
+      width: '15%',
+      key: 'projects',
+      render: () => {
+        return organization ? (
+          <div className="projects">Projects: {organization.projects.length}</div>
+        ) : (
+          <>Projects -</>
+        );
+      },
+    },
```

Comment:
Displaying the projects the user is associated to within the context of an organization is not really great to do. Since users are assigned to groups, the reference of which projects they are in is pulled from these groups. I think it would be best to omit project references to users for now because of the complexity around actually collecting this information.

Displaying it to people also seems to imply that someone could theoretically remove a user from a project specifically, which is not actually possible as projects are assigned to groups like users are, so you can't simply remove a user from a project if the project and user can be attached/shared amongst multiple groups.